lib/ui/contact_list/chat_list_screen.dart

```diff
@@ -219,6 +222,13 @@ class _ChatListScreenState extends ConsumerState<ChatListScreen> with TickerProv
                   (item.lastMessage?.content?.toLowerCase().contains(searchLower) ?? false);
             }).toList();
 
+    final noRelayConnected = ref
```

Comment:
I have a doubt... I guess that as relays are for different things, with a single relay in one kind, some features will stil not work... wouldn't me more useful to check for at least one relay of each kind?

Reply:
Other question... maybe it would be easier to add this method to the relays provider Instead of adding more logic to the screen ... what do you think?